libraries/scripting/type-providers/src/kotlin/script/experimental/typeProviders/generatedCode/impl/makeInlineClass.kt

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
```

Comment:
I guess it should belong to a different commit.

Reply:
Oh yeah. I'll redo the entire git history after we our call

Reply:
Redid history